.travis.yml

```diff
@@ -1,49 +1,73 @@
 language: c
-sudo: false
+sudo: require
 
 matrix:
   include:
   - os: linux
-    compiler: gcc
-    env: CC_OQS=gcc-4.8
+    compiler: gcc-4.8
+    env: CC=gcc-4.8
+    before_install:
+      - sudo apt-get -qq update
+      - sudo apt-get install -y autotools-dev
+      - sudo apt-get install --only-upgrade -y autoconf 
     addons:
       apt:
         sources:
         - ubuntu-toolchain-r-test
         packages:
         - gcc-4.8
   - os: linux
-    compiler: gcc
-    env: CC_OQS=gcc-4.9
+    compiler: gcc-4.9
+    env: CC=gcc-4.9
+    before_install:
+      - sudo apt-get -qq update
+      - sudo apt-get install -y autotools-dev
+      - sudo apt-get install --only-upgrade -y autoconf 
     addons:
       apt:
         sources:
         - ubuntu-toolchain-r-test
         packages:
         - gcc-4.9
   - os: linux
-    compiler: gcc
-    env: CC_OQS=gcc-5
+    compiler: gcc-5
+    env: CC=gcc-5
+    before_install:
+      - sudo apt-get -qq update
+      - sudo apt-get install -y autotools-dev
+      - sudo apt-get install --only-upgrade -y autoconf 
     addons:
       apt:
         sources:
         - ubuntu-toolchain-r-test
         packages:
         - gcc-5
+        - autotools-dev
+        - autoconf
   - os: linux
-    compiler: gcc
-    env: CC_OQS=gcc-6
+    compiler: gcc-6
+    env: CC=gcc-6
+    before_install:
+      - sudo apt-get -qq update
+      - sudo apt-get install -y autotools-dev
+      - sudo apt-get install --only-upgrade -y autoconf 
     addons:
       apt:
         sources:
         - ubuntu-toolchain-r-test
         packages:
         - gcc-6
+        - autotools-dev
+        - autoconf
   - os: osx
     compiler: clang
-    env: CC_OQS=clang
+    env: CC=clang
+    before_install:
+      -  brew install autoconf
+      -  brew install automake
   
-
 script:
-  - make
-  - make check
\ No newline at end of file
+  - autoreconf -i 
+  - ./configure  
```

Comment:
Are you sure the ./configure is applied with the right compiler everytime? Can we do something like 
`./configure CC=... `
?